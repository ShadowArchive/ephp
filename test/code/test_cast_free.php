<?php

$a = [( int ) "100", (INT) "200", ( Int ) "300"];

var_dump($a);
