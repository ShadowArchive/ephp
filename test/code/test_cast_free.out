array(3) {
  [0]=>
  int(100)
  [1]=>
  int(200)
  [2]=>
  int(300)
}
